_studio/shared/src/libmfx_core_vaapi.cpp

```diff
@@ -424,6 +424,22 @@ VAAPIVideoCORE::GetHandle(
     MFX_CHECK_NULL_PTR1(handle);
     UMC::AutomaticUMCMutex guard(m_guard);
 
+#if defined (MFX_ENABLE_CPLIB)
+#if (MFX_VERSION >= 1030)
```

Comment:
I suggest to drop check for MFX_VERSION here and below where it is paired with MFX_ENABLE_CPLIB. Just make sure that we define MFX_ENABLE_CPLIB only if MFX_VERSION>=1030.

Reply:
fixed